signer/src/bitcoin/validation.rs

```diff
@@ -201,15 +479,14 @@ impl BitcoinDepositInputError {
 
 /// The responses for validation of the outputs of a sweep transaction on
 /// bitcoin.
-#[derive(Debug, thiserror::Error, PartialEq, Eq, Copy, Clone)]
-pub enum BitcoinWithdrawalOutputError {
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+pub enum WithdrawalValidationResult {
     /// The signer does not have a record of the withdrawal request in
     /// their database.
-    #[error("the signer does not have a record of the withdrawal request; {}", .0.request_id)]
-    Unknown(QualifiedRequestId),
+    Unsupported,
```

Comment:
is this `Unsopported`  instead of `Unknown` to indicate that we don't do support withdrawals as of now? I was confused by the comment above

Reply:
Oh yeah, I should update this

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/849/commits/bd9d9ebb4094dd06f3f3f7c47ea46e12ca668363